### DISCUSSION TITLE
fibers.js missing or malformed argument

### DISCUSSION THREAD

**Discussion by CornBread81032133** [kromitgmbh/titra#102] [Q&A]
Hi everyone,

I've recent come across Titra and wanted to spin up a container to see if it suits my work needs.
On deploying the container, I'm getting an error that an argument is either missing or malformed, in fibers.js? 
Didn't see anything like this here, so I was hoping to get some advice.

```
/app/bundle/programs/server/node_modules/fibers/fibers.js:92
					return fn.apply(this, arguments);
					          ^
Error: Must pass options.rootUrl or set ROOT_URL in the server environment
    at Object.Meteor.absoluteUrl (packages/meteor.js:1412:11)
    at runWebAppServer (packages/webapp/webapp_server.js:996:48)
    at packages/webapp/webapp_server.js:1494:1
    at module (packages/webapp/webapp_server.js:1494:16)
    at fileEvaluate (packages/modules-runtime.js:336:7)
    at Module.require (packages/modules-runtime.js:238:14)
    at require (packages/modules-runtime.js:258:21)
    at /app/bundle/programs/server/packages/webapp.js:1963:15
    at /app/bundle/programs/server/packages/webapp.js:1972:3
    at /app/bundle/programs/server/boot.js:401:38
    at Array.forEach (<anonymous>)
    at /app/bundle/programs/server/boot.js:226:21
    at /app/bundle/programs/server/boot.js:464:7
    at Function.run (/app/bundle/programs/server/profile.js:280:14)
    at /app/bundle/programs/server/boot.js:463:13
```

**Comment by faburem** [comment]
Hi,
as you can see in the error message you must pass the ROOT_URL to the container. In general you will also need a mongodb container for the database backend so I would recommend to go with the provided docker-compose.yml:
https://github.com/kromitgmbh/titra/blob/056c6502bd070bfd2678069dc5d4b8a431565030/docker-compose.yml#L9

The easiest way to get started is mentioned on the readme of the github repo:
`curl -L https://github.com/kromitgmbh/titra/raw/master/docker-compose.yml | ROOT_URL=http://localhost:3000 docker-compose -f - up
`

Best,
Fabian

**Reply by CornBread81032133** [reply]
Hi Dabian,
This was deployed with the mentioned method in the GitHub repo home page. 
Based on the above, it sounds like it's something I'm doing wrong, or within my environment, so I'll investigate what that is.
Thanks for the assistance.
